keyboards/wookong/wk87/keymaps/via/keymap.c

```diff
@@ -0,0 +1,81 @@
+/* Copyright 2023 Gyphae Team <https://github.com/gyphae>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// Copyright 2020 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Shouldn't mix 2 types of licence header.

Reply:
Done